src/ConferenceItem.jsx

```diff
@@ -1,20 +1,40 @@
 import React from "react";
-import moment from "moment";
+import styled from "styled-components";
+
+const StyledCard = styled.div`
```

Comment:
Nice intro to styled components, if you wanted to go a step further you could pass props to the card and then render a different background colour for events that start in the morning or night ( This would only be for your benefit to understand how to render depending on props, and not part of the ticket, but would be useful imo )

Reply:
We just had a play around with this and got it to work, didn't include it in the latests commit though since it was for practice.

Reply:
At this stage theres no harm in including it, as helpers may find a better way to write it as no doubt in the future you will be passing props at some point, similarly it'll help the other guys see how to do it when its their turn to see! but its cool dont worry about it :)

Reply:
Good point, we've added them in.